ethpm/assets/package.schema.v2.json

```diff
@@ -163,6 +166,9 @@
         "block": {
           "$ref": "#/definitions/BlockHash"
         },
+		"bytecode": {
```

Comment:
Is it safe to assume that if `bytecode` is present, then there will always be a `runtime_bytecode` object present (and vice versa). I know with jsonschema we can add dependencies that can enforce that